GoMeet/src/main/webapp/js/dao-utils.js

```diff
@@ -0,0 +1,21 @@
+class DAOUtils {
+  /**
+   * Returns a URL string with the key value pairs
+   * as given in 'keyValues' encoded into a query string, in the format
+   * '/endpoint?key1=value1&key2=value2...'
+   * @param {String} endpoint the endpoint of the URL, that is, the part of the URL 
+   * just before the query string INCLUDING the '?', associated with the relevant servlet
```

Comment:
I might leave this for now and make this change in a separate branch as it would involve refactoring Anna and Therese's code too (since they also use this function)